packages/nimble-components/src/list-option/index.ts

```diff
@@ -66,6 +66,13 @@ export class ListOption extends FoundationListboxOption {
         super.connectedCallback();
         const owner = this.getListOptionOwner();
         owner?.registerOption(this);
+        const observer = new MutationObserver(_ => {
+            this.$emit('display-changed');
```

Comment:
I'm curious if you considered the alternative to emitting an event here of making `elementTextContent` observable. And if so, what the rationale was for choosing an event instead of an observable.